oem/ibm/libpldmresponder/file_io_type_cert.hpp

```diff
@@ -51,7 +51,7 @@ class CertHandler : public FileHandler
 
     virtual int newFileAvailable(uint64_t length);
 
-    virtual int fileAckWithMetaData(uint8_t /*fileStatus*/,
+    virtual int fileAckWithMetaData(uint8_t fileStatus,
```

Comment:
why this change? if change is required then cert.cpp file is not present for review